localstack/services/awslambda/lambda_api.py

```diff
@@ -2240,3 +2240,29 @@ def serve(port):
     LAMBDA_EXECUTOR.startup()
 
     generic_proxy.serve_flask_app(app=app, port=port)
+
+
+# Config listener
+def on_config_change(config_key: str, config_newvalue: str) -> None:
+    global LAMBDA_EXECUTOR
+    if config_key != "LAMBDA_EXECUTOR":
+        return
+    LOG.debug(
+        "Recieved config event for lambda executor! Key: {} Value: {}".format(
```

Comment:
nitpick: `Recieved` -> `Received` (also, I'd probably avoid using exclamation marks in (debug) log messages).